.all-contributorsrc

```diff
@@ -275,6 +275,13 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "wabirached",
+      "name": "Wissam Abirached",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/3534924?v=3",
+      "profile": "https://designingforscale.com",
+      "contributions": []
```

Comment:
Do you want to add `code` and `test` to this? Then run `npm start generateContributors` to regenerate the table :+1: